bloomberg/xcdiff
# Add project attributes comparator

*Issue number of the reported bug or feature request: #62*

**Describe your changes**

- Added a new attributes comparator to compare project attributes (such as organization name & last upgrade check version)


**Testing performed**

- Run `swift run xcdiff --tag attributes --verbose` withing `Fixtures`
- Verify the results include project attributes (like organization name)

e.g.

```
❌ ATTRIBUTES > Root project

⚠️  Value mismatch (3):

  • LastSwiftUpdateCheck
    ◦ 1110
    ◦ 1030

  • LastUpgradeCheck
    ◦ 1020
    ◦ 1030

  • ORGANIZATIONNAME
    ◦ Bloomberg LP
    ◦ Another Organization
```
